arangod/VocBase/Methods/Indexes.cpp

```diff
@@ -740,6 +740,7 @@ Result Indexes::drop(LogicalCollection* collection,
                                         collection->vocbase(), false),
                                     *collection, AccessMode::Type::EXCLUSIVE,
                                     trxOpts);
+    trx.addHint(transaction::Hints::Hint::GLOBAL_MANAGED);
```

Comment:
IMO this hint is not necessary here, because the operations (dropping the index) will only be applied locally. This is not sending any requests out and is not replicated.